lib/signs/supervisor.ex

```diff
@@ -19,9 +19,15 @@ defmodule Signs.Supervisor do
   weird cases?)
   """
   require Signs.Utilities.SignsConfig
+  use Supervisor
 
-  def start_link do
-    Supervisor.start_link(children(), name: __MODULE__, strategy: :one_for_one)
+  def start_link([]) do
+    Supervisor.start_link(__MODULE__, name: __MODULE__, strategy: :one_for_one)
```

Comment:
😮 thank you for calling this out.

I added some logging to list out all of the child processes of this supervisor and oddly enough all of the processes for each of the 234 signs seem to start up with no problems so that is kinda weird. But I updated the syntax to reflect what's in Hexdocs since that's probably the safer thing to do.